app/src/main/java/org/simple/clinic/home/report/ReportsEvent.kt

```diff
@@ -1,9 +1,13 @@
 package org.simple.clinic.home.report
 
 import org.simple.clinic.util.Optional
+import org.simple.clinic.widgets.UiEvent
 
-sealed class ReportsEvent
+sealed class ReportsEvent : UiEvent
 
 data class ReportsLoaded(val reportsContent: Optional<String>) : ReportsEvent()
 
-object WebBackClicked : ReportsEvent()
+object WebBackClicked : ReportsEvent() {
+
+  override val analyticsName = "Reports Back clicked"
```

Comment:
Can we follow the same convention as the rest of the screens: "<Screen Name>:<Event name>" please?